### PR TITLE
Add dribbble shots to the homepage

### PR DESCRIPTION
**Pull request by creuter**
screencapture-localhost-4000-1464185005502

### CODE REVIEW

**Comment by tysongach** [comment]
👍 I’m no JavaScript person, but looks good!

File: web/static/css/_sections.scss [hunk 1]
```diff
@@ -1,22 +1,19 @@
 .section {
-  margin-bottom: $large-spacing * 2;
+  border-top: $base-border;
+  padding-top: $large-spacing;
+  margin-bottom: $x-large-spacing;
```
Comment (jaclynperrone) [inline web/static/css/_sections.scss:4]: 🔤

Comment (creuter) [inline web/static/css/_sections.scss:4]: Whoops, I just missed this. I'll knock it out in the PR. Thank you!

**Comment by jaclynperrone** [comment]
LGTM! 🌻

**Pull request merged**